libraries/AP_HAL_PX4/RCOutput.cpp

```diff
@@ -329,7 +331,12 @@ void PX4RCOutput::write(uint8_t ch, uint16_t period_us)
         _max_channel = ch + 1;
     }
 
+
     if (_output_mode == MODE_PWM_BRUSHED) {
+
+        // keep unscaled value
+        _last_sent[ch] = period_us;
```

Comment:
This shouldn't be inside this if, otherwise the returned value for `read_last_sent` when not using brushed motors will be wrong.